project/design/schema.sql

```diff
@@ -0,0 +1,59 @@
+// TO DISTINGUISH BETWEEN ADMIN OR NORMAL USER
+CREATE TABLE IF NOT EXISTS usertype(
+	type varchar(20) PRIMARY KEY
+	);
+
+CREATE TABLE IF NOT EXISTS user(
+
```

Comment:
good to have id, so you can use get by id, update by id and so on  methods on id.

Reply:
done